packages/mediacenter/kodi-binary-addons/visualization.matrix/package.mk

```diff
@@ -0,0 +1,27 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
```

Comment:
```suggestion
# SPDX-License-Identifier: GPL-2.0
```
The LE code is under GPL2.0 so we obviously prefer to use it at new packages too.